Makefile

```diff
@@ -114,8 +115,13 @@ node_modules/.uptodate: package.json
 .PHONY: help
 help:
 	@echo "The following targets are available:"
+	@echo
 	@echo " clean      Clean up runtime artifacts (needed after a version update)"
 	@echo " dev        Run the development H server locally"
+	@echo " dist       Build the python H distribution"
 	@echo " docker     Build hypothesis/hypothesis docker image"
 	@echo " extensions Build the browser extensions"
 	@echo " test       Run the test suite (default)"
+	@echo
+	@echo "To run the test suite and build the whole app, you can do:"
+	@echo " $ make clean && make && make dist && make extensions"
```

Comment:
Did you see this comment? I'm happy to merge this but was waiting for you to update this.